docs/ExpectAPI.md

```diff
@@ -1209,9 +1211,13 @@ test('throws on octopus', () => {
 
 > Note: You must wrap the code in a function, otherwise the error will not be caught and the assertion will fail.
 
-### `.toThrowErrorMatchingSnapshot()`
+### `.toThrowErrorMatchingSnapshot(snapshotName?)`
+
+Use `.toThrowErrorMatchingSnapshot` to test that a function throws an error matching the most recent snapshot when it is called.
+
+You can provide an optional `snapshotName` string argument for the snapshot key, in addition to the concatenated name arguments of containing blocks. Jest also appends `1`, `2`, `3`, and so on to snapshot keys, in case an `it` or `test` block contains multiple snapshot assertions.
```

Comment:
I find this description convoluted around "snapshot key" and "nam arguments of containing blocks". Can we make it simpler?

Reply:
Yes, thank you. What do you think of this rewrite?

You can provide an optional `snapshotName` string argument for the snapshot key. It follows the concatenated name arguments of containing blocks and precedes the final number (that Jest appends in case an `it` or `test` block contains multiple snapshot assertions).

I deleted the comment that I wrote in the main thread by mistake.

Reply:
s/multiple snapshot assertions/more than one snapshot assertion/

Reply:
How about:

You can provide an optional `snapshotName` string argument that will be appended to the corresponding test title in `.snap` file, followed by a number that Jest appends to differentiate between snapshots created during single `it` or `test` block.

(the number is always appended)

Reply:
Oh btw, can you mention that snapshots are sorted in alphabetical order for stability? I don't see this mentioned anywhere in the docs